applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -220,6 +226,11 @@ int lwm2m_carrier_event_handler(const lwm2m_carrier_event_t *event)
 
 	rsp_send("\r\n#XCARRIEREVT: %d,%d\r\n", event->type, err);
 
+	/* Allow time for the URC be flushed before reboot */
+	if (event->type == LWM2M_CARRIER_EVENT_REBOOT && err == 0) {
+		k_sleep(K_MSEC(100));
```

Comment:
SBM asked for a delay of 5s and I offered 100ms as I'm not sure about context of callback in LwM2M stack. SLM_UART_RESPONSE_DELAY is 50ms.

Reply:
`SLM_UART_RESPONSE_DELAY` could be increased to 100ms if you think 100ms is needed. I was thinking it'd be nice to use it here because it's for the same purpose than other places using it.

Reply:
@tomi-font SBM just accepted current behavior without further requesting time (5min) for host side to prepare for a reboot. So I'll silently change the delay from 100ms to 50ms in next submit.